src/pyroboplan/planning/prm.py

```diff
@@ -67,8 +73,12 @@ class PRMPlanner:
 
     Graphs can be persisted to disk for use in future applications.
 
-    Link to the original publication:
-        https://www.kavrakilab.org/publications/kavraki-svestka1996probabilistic-roadmaps-for.pdf
+    Some helpful resources:
+        * The original publication:
+          https://www.kavrakilab.org/publications/kavraki-svestka1996probabilistic-roadmaps-for.pdf
+        * A nice guide to higher dimensional motion planning:
+          https://motion.cs.illinois.edu/RoboticSystems/MotionPlanningHigherDimensions.html
```

Comment:
This might be good to drop in the docs I pointed to in the top comment too.

Reply:
Added! Hopefully I can still write sensible English...